pkg/ddl/create_table.go

```diff
@@ -180,11 +182,12 @@ func onCreateTable(jobCtx *jobContext, t *meta.Meta, job *model.Job) (ver int64,
 
 	// Finish this job.
 	job.FinishTableJob(model.JobStateDone, model.StatePublic, ver, tbInfo)
-	createTableEvent := statsutil.NewCreateTableEvent(
-		job.SchemaID,
-		tbInfo,
-	)
-	asyncNotifyEvent(jobCtx, createTableEvent)
+	if !tidb_util.IsMemOrSysDB(job.SchemaName) {
```

Comment:
Not sure if this is a good idea because maybe we should only handle DDL events. It seems like consumer behavior.

Reply:
I slightly prefer we filter it at producer. Because the modification of these DBs are done at bootstrap, it's more clear to let consumer nodes also use bootstrap phase to handle it, rather than using the pub-sub framework.

Reply:
Or can we do it within asyncNotifyEvent? That would help us to avoid writing it again and again.

Reply:
https://github.com/pingcap/tidb/pull/55811/commits/a0a25a42f4dac2d9f50a9d3f92ae952fbd38be2a